broadcaster/lib/broadcaster/forwarder.ex

```diff
@@ -116,6 +197,16 @@ defmodule Broadcaster.Forwarder do
     {:reply, :ok, %{state | pending_outputs: pending_outputs}}
   end
 
+  @impl true
+  def handle_cast({:send_pli, id, layer}, state) do
+    with {:ok, input_pc, input} <- find_input(id, state),
+         true <- is_pid(input_pc) do
+      PeerConnection.send_pli(input_pc, input.video, layer)
+    end
+
+    {:noreply, state}
+  end
```

Comment:
Used `cast` here since RPC calls are expensive and we don't really need to confirm (or block on) sending a PLI